master/pkg/check/check_string.go

```diff
@@ -30,3 +31,42 @@ func Match(actual string, regex string, msgAndArgs ...interface{}) error {
 	return check(compiled.FindString(actual) == actual, msgAndArgs,
 		"%s doesn't match regex %s", actual, regex)
 }
+
+// LenBetween checks whether the length of the first argument is between the second and third arguments.
+// The method returns an error with the provided message if the check fails.
+func LenBetween(actual string, min, max int, msgAndArgs ...interface{}) error {
+	return BetweenInclusive(len(actual), min, max, msgAndArgs...)
+}
+
+// IsValidK8sLabel checks whether the first argument is a valid Kubernetes label. The method returns
+// an error with the provided message if the check fails.
+func IsValidK8sLabel(actual string, msgAndArgs ...interface{}) error {
+	if err := NotEmpty(actual, msgAndArgs...); err != nil {
+		return err
+	}
+	if err := LenBetween(actual, 1, 63, msgAndArgs...); err != nil {
+		return err
+	}
+	if err := Match(actual, `^[a-zA-Z0-9]([-a-zA-Z0-9_.]*[a-zA-Z0-9])?$`, msgAndArgs); err != nil {
+		return err
+	}
+	return nil
+}
+
+// IsValidIPV4 checks whether the first argument is a valid IPv4 address. The method returns an error
+// with the provided message if the check fails.
+func IsValidIPV4(actual string, msgAndArgs ...interface{}) error {
```

Comment:
is there a stdlib function that does this (also might not be needed if we allow hostnames too)